scene/3d/lightmap_gi.cpp

```diff
@@ -354,7 +355,16 @@ void LightmapGI::_find_meshes_and_lights(Node *p_at_node, Vector<MeshesFound> &m
 	Node3D *s = Object::cast_to<Node3D>(p_at_node);
 
 	if (!mi && s) {
-		Array bmeshes = p_at_node->call("get_bake_meshes");
+		Array bmeshes;
+		MultiMeshInstance3D *multi_mesh = Object::cast_to<MultiMeshInstance3D>(p_at_node);
+		if (multi_mesh) {
+			// By default, get_meshes returns an Array with the transform first where the code
+			// below expects the mesh first.  Request the order swapped so baking works.
+			bmeshes = multi_mesh->get_meshes(true);
```

Comment:
Instead of passing a parameter to get_meshes, I would just swap the order here. Since this function is the one with unique expectations, its better to keep the complexity here, instead of spreading it to MultiMeshInstance